guides/common/modules/proc_configuring-minimal-data-collection-for-hosts.adoc

```diff
@@ -0,0 +1,38 @@
+[id="configuring-minimnal-data-collection-for-hosts"]
+= Configuring Minimal data collection for hosts
```

Comment:
I'm not happy with this title. How does this sound?

```suggestion
= Limiting collected data to a minimum
```
Feel free to suggest another or rephrase.